### DISCUSSION TITLE
How to remove Core Plugin?

### DISCUSSION THREAD

**Discussion by FahimAnayet** [LunarVim/LunarVim#3071] [Q&A] [answered]
How to remove core plugin of lvim? I want to remove some core plugins that I will never use. How can I accomplish that?
Thanks.

**Comment by latipun7** [comment, marked as answer]
currently you could disable that with `lvim.builtin.<name_of_plugin_or_module>.active = false`

for example, if you want to disable navic, it would be `lvim.builtin.breadcrumbs.active = false`

for what plugin you could disable, see this file: https://github.com/LunarVim/LunarVim/blob/257e3ff3697a55e7a17eab876fb8d0b85fd203de/lua/lvim/plugins.lua#L43
